modules/sspBCBidAdapter.js

```diff
@@ -104,6 +104,7 @@ const getNotificationPayload = bidData => {
 
         // basic notification data
         const bidBasicData = {
+          // TODO: fix auctionId leak: https://github.com/prebid/Prebid.js/issues/9781
```

Comment:
they define a fallback, seems okay to replace and leave their fallback in place?